### PR TITLE
make music asset default landing page for admin and music provider

### PR DESCRIPTION
**Pull request by jvmsangkal**
## A. Summary of Changes
Change login redirect to assets lists

### a. DB Changes:
- N/A

### b. Package Changes:
- none

### c. Implementation Changes:
- Login callback to redirect to asset lists
- Change nav dropdown to asset lists as well

### d. Other things to do:
- none

### a. Ticket link/Request Feature link
- [Tunes-109](https://freedom.myjetbrains.com/youtrack/issue/Tunes-109)

### b. Changed files/directories:
- See diffs

### c. Tests Done
- [Functional Test in Asana](https://app.asana.com)

### CODE REVIEW

File: app/scripts/components/front/login/callback.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ var React = require('react'),
                     return self.context.router.push('contract');
                 }
 
+                if (user_info.user_info.role === 'admin') {
+                    return self.context.router.push('/admin');
```
Comment (sibayanjasper) [inline app/scripts/components/front/login/callback.js]: Should it be the case? Upon logging in, admin should still see the front page.
If you do this, "Use as admin" menu will make no sense.

Comment (rycastaneda) [inline app/scripts/components/front/login/callback.js]: You can still go back to home use as regular user then use as admin so its still useful

Comment (sibayanjasper) [inline app/scripts/components/front/login/callback.js]: Got it.
Meanwhile, change `user_info` to a more appropriate name. `user_info.user_info` does not sound good.

Comment (jvmsangkal) [inline app/scripts/components/front/login/callback.js]: okay

**Pull request merged**